Library/Homebrew/cmd/tap-info.rb

```diff
@@ -69,7 +69,6 @@ def print_tap_info(taps)
         puts unless i.zero?
         info = "#{tap}: "
         if tap.installed?
-          info += tap.pinned? ? "pinned" : "unpinned"
           info += ", private" if tap.private?
           info += if (contents = tap.contents).empty?
             ", no commands/casks/formulae"
```

Comment:
Should probably handle the comma better here. Currently outputs as:

```
homebrew/core: , 2 commands, 5166 formulae
```

Reply:
Good catch, https://github.com/Homebrew/brew/pull/8236